testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkIcebergRest.java

```diff
@@ -43,7 +43,8 @@
     private static final int REST_SERVER_PORT = 8181;
     private static final String SPARK_CONTAINER_NAME = "spark";
     private static final String REST_CONTAINER_NAME = "iceberg-with-rest";
-    private static final String REST_SERVER_IMAGE = "tabulario/iceberg-rest:0.2.0";
+    // FIXME: remove before merging
+    private static final String REST_SERVER_IMAGE = "tabulario/iceberg-rest:0.3.0-snapshot";
```

Comment:
once 1.2.0 is officially out, we'll release `tabulario/iceberg-rest:0.3.0`

Reply:
this can be updated now to `tabulario/iceberg-rest:0.3.0`

Reply:
Thanks, updated